api/src/test/resources/io/strimzi/api/kafka/model/KafkaUser.yaml

```diff
@@ -13,18 +13,18 @@ spec:
           type: topic
           name: my-topic
           patternType: prefix
-        operation: Read
+        operations: ["Read","Write"]
       - resource:
           type: topic
           name: my
           patternType: prefix
-        operation: Read
+        operations: ["Read","Describe","Write","Create"]
         host: 127.0.0.1
       - resource:
           type: group
           name: my-group
-        operation: Read
+        operations: ["Read"]
```

Comment:
I think you should have a test which tests the `operation` field as well. It is still there so it should be tested.